.ci/integration-tests/setup-tpm2provisioner.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Script to setup the TPM2 Provisioner Docker Image for Integration Tests
+
+set -e
+
+# Wait for ACA to boot
+until [ "`curl --silent --connect-timeout 1 -I -k https://localhost:8443/HIRS_AttestationCAPortal | grep '302 Found'`" != "" ]; do
+    :
+done
+
+pushd /HIRS
+if [ ! -d package/rpm/RPMS ]; then
+    ./package/package.centos.sh
+fi
+yum install -y package/rpm/RPMS/x86_64/HIRS_Provisioner_TPM_2_0*.el7.x86_64.rpm
+popd
+
+mkdir -p /var/run/dbus
+if [ -e /var/run/dbus/pid ]; then
+  rm /var/run/dbus/pid
+fi
+
+if [ -e /var/run/dbus/system_bus_socket ]; then
+  rm /var/run/dbus/system_bus_socket
+fi
+
+# Start the DBus
+dbus-daemon --fork --system
+echo "DBus started"
+
+# Give DBus time to start up
+sleep 5
+
+/ibmtpm/src/./tpm_server &
+echo "TPM Emulator started"
+
+tpm2-abrmd -t socket &
+echo "TPM2-Abrmd started"
+
+# Give ABRMD time to start and register on the DBus
+sleep 5
+
+# EK and PC Certificate
+ek_cert_der="/HIRS/.ci/integration-tests/certs/ek_cert.der"
+platform_cert="platformAttributeCertificate.pem"
+
+echo "Creating Platform Cert for Container"
+PC_DIR=/var/hirs/pc_generation
+mkdir -p $PC_DIR
+/opt/paccor/scripts/allcomponents.sh > $PC_DIR/componentsFile
+/opt/paccor/scripts/referenceoptions.sh > $PC_DIR/optionsFile
+/opt/paccor/scripts/otherextensions.sh > $PC_DIR/extensionsFile
+/opt/paccor/bin/observer -c $PC_DIR/componentsFile -p $PC_DIR/optionsFile -e $ek_cert_der -f $PC_DIR/observerFile
+/opt/paccor/bin/signer -o $PC_DIR/observerFile -x $PC_DIR/extensionsFile -b 20180101 -a 20280101 -N $RANDOM -k /HIRS/.ci/integration-tests/certs/ca.key -P /HIRS/.ci/integration-tests/certs/ca.crt --pem -f $PC_DIR/$platform_cert
+
```

Comment:
Think it's just an emulator issue, but you may get errors loading the the certs into NVRam. Checking and releasing it seems to fix the problem:

```suggestion
 	if tpm2_nvlist | grep -q 0x1c00002; then
	  tpm2_nvrelease -x 0x1c00002 -a 0x40000001 
	fi
```

Reply:
A couple of us had this issue using the 2.0 Emulator on our linux boxes.
And I had the same problem when testing with the Docker images.
I figured it wouldn't be a problem with the fresh installs. Or maybe I'm just messing something up somehow...

Reply:
Yeah I'm not sure that any of us over here ran into that issue, and I haven't done a deep-dive on all the changes you've made on your branch. It looked like you did a lot of cleaning and refinement of this initial cut, so it's possible something shifted in transit?

Reply:
Done.